README.md

```diff
@@ -88,9 +88,10 @@ Thanks to all [contributors](https://github.com/testthedocs/awesome-docs/graphs/
 
 ## Editor
 
+- [HackMD](https://hackmd.io/)
```

Comment:
Moved to be sorted Alphabetically - according to the guidelines

Reply:
Good job